tests/test_swh_api_functions.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import os
+import json
+import pytest
+from src import swh_api_functions
+
+# class to genrate our own custom response
+from custom_api_response import CustomApiResponse
+
+BASE_SWH_API_URL = "https://archive.softwareheritage.org/api/1/"
+
+
+def test_get_content_information(requests_mock):
+    # input data
+    hash_type = "sha1_git"
+    hash = "fe95a46679d128ff167b7c55df5d02356c5a1ae1"
+    # output data
+    json_output = {
+        "checksums": {
```

Comment:
All these should be the fields defined on an object.

Reply:
Sorry but i didn't get it??. Do you mean like i shoud have an object like input_obj and then set these attributes to it??
Like this {checksum_type:' ', checksum: ' '} ??

Reply:
@sanketvarpe I am only restating exactly the same thing that @TG1999 said earlier in https://github.com/nexB/heritedcode/pull/1#discussion_r796337937 with different words: 

> Instead of returning a json we should return some type of object which we can obtain by mapping json with some kind of class object, since it returns checksums, data_url, file_url, license_url. We should create a class with at least these as attributes for now.